### DISCUSSION TITLE
关于添加密码

### DISCUSSION THREAD

**Discussion by yehuzhucha** [spencerwooo/onedrive-vercel-index#403] [Q&A] [answered]
修改要保护的文件夹及上传.password文件后页面报错
![image](https://user-images.githubusercontent.com/70362459/153389341-60e8ed11-a860-4726-b226-2ba27b3aa1d3.png)
麻烦请问大佬应如何解决

**Comment by spencerwooo** [comment, marked as answer]
Check Vercel function logs?

**Reply by yehuzhucha** [reply]
已解决，换了个编辑器成功了
